test/parallel/test-child-process-spawnsync-maxbuf.js

```diff
@@ -16,7 +16,8 @@ const args = [
 {
   const ret = spawnSync(process.execPath, args, { maxBuffer: 1 });
 
-  assert.ok(ret.error, 'maxBuffer should error');
+  assert.ok(ret.error, 'child process does not error \
+when maxBuffer is set to 1');
```

Comment:
this your last commit introduced non-standard indentations. Please will you correct it? The previous commit was fair enough, dont know why you pushed again

Reply:
thanks for the review. Sorry, that I have to change it again. I only wanted to fix the error that the linter was throwing, where i exceeded the maximum line length of 80.